fea-rs/src/compile/lookups.rs

```diff
@@ -182,7 +183,8 @@ pub(crate) struct LookupFlagInfo {
 }
 
 /// A feature associated with a particular script and language.
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash, PartialOrd, Ord)]
+#[derive(Clone, Copy, PartialEq, Eq, Hash, PartialOrd, Ord)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 pub struct FeatureKey {
```

Comment:
Worth considering types for feature vs language vs script? - it appears easy to confuse them under the current definition.

Reply:
my feeling is that at this point the types would be more noise than help, but if it ends up being a problem we can always go that route.